Automattic/wp-calypso
# WP for Teams: hide non-relevant stuff from Calypso

Fixes https://github.com/Automattic/wp-for-teams/issues/32 and https://github.com/Automattic/wp-for-teams/issues/33.

As part of our WP for Teams project, we need to hide a few menus, banners (until we have a pricing and plan strategy ready) and other items. This applies only to WP for Teams sites — others are unaffected.

## Testing instructions

Go over all the mentioned points in the linked issue to verify they are hidden for a WP for Teams site. Make sure to test a non WP for Team sites if they have all the items.

You can create a new WP for Teams site through the `start/wp-for-teams` signup flow (working locally and on staging).
